pydarn/plotting/rtp.py

```diff
@@ -26,22 +29,602 @@ class RTP(matplotlib.pyplot):
 
     Methods
     -------
-    plot
-    summaryplot
+    plot_profile
+    plot_scalar
+    summary_plot
 
     """
+    # Default parameter types that davitpy supports and are usually involved
+    # in a summary plot
+    parameter_type = {'power': ('p_l', r'Signal to Noise ($dB$)'),
+                      'velocity': ('v', r'Velocity ($m/s$)'),
```

Comment:
Recommend changing $m/s$ to $m s^{-1}$

Reply:
I removed this after your request in PR #9; however, the change did not propagate. So I merged the `rtp\plot_range_time` into `rtp\plot_time_series`.